src/constants.ts

```diff
@@ -8,18 +8,19 @@
 
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
+import fs from 'fs';
+import path from 'path';
 
-exports.MessageType = {};
+export const MessageType = {};
 
 // Build up message types dynamically from schemas
 const dname = path.resolve(path.join(__dirname, '..', 'schema', 'messages'));
+
 for (const fname of fs.readdirSync(dname)) {
-  const schema = JSON.parse(fs.readFileSync(path.join(dname, fname)));
+  const schema = JSON.parse(fs.readFileSync(path.join(dname, fname)).toString());
 
   if (!schema.hasOwnProperty('properties') ||
-      !schema.properties.hasOwnProperty('messageType')) {
+    !schema.properties.hasOwnProperty('messageType')) {
```

Comment:
nit: why the spacing change?

Reply:
The default formatter did this.
It should be fixed with the last commit.